preferencesfx/src/main/java/com/dlsc/preferencesfx/util/DefaultStorageHandler.java

```diff
@@ -0,0 +1,243 @@
+package com.dlsc.preferencesfx.util;
+
+import static com.dlsc.preferencesfx.util.Constants.DEFAULT_DIVIDER_POSITION;
+import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_HEIGHT;
+import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_POS_X;
+import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_POS_Y;
+import static com.dlsc.preferencesfx.util.Constants.DEFAULT_PREFERENCES_WIDTH;
+import static com.dlsc.preferencesfx.util.Constants.DIVIDER_POSITION;
+import static com.dlsc.preferencesfx.util.Constants.SELECTED_CATEGORY;
+import static com.dlsc.preferencesfx.util.Constants.WINDOW_HEIGHT;
+import static com.dlsc.preferencesfx.util.Constants.WINDOW_POS_X;
+import static com.dlsc.preferencesfx.util.Constants.WINDOW_POS_Y;
+import static com.dlsc.preferencesfx.util.Constants.WINDOW_WIDTH;
+
+import com.dlsc.preferencesfx.model.Setting;
+import com.google.gson.Gson;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.prefs.BackingStoreException;
+import java.util.prefs.Preferences;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Handles everything related to storing values of {@link Setting} using {@link Preferences}.
+ *
+ * @author François Martin
+ * @author Marco Sanfratello
+ */
+public class DefaultStorageHandler implements StorageHandler {
```

Comment:
I think "StorageHandlerImpl" would be better suited as a name, since this is a convention that is often used for default implementations of interfaces